src/sentry/db/models/base.py

```diff
@@ -74,7 +75,7 @@ def _update_tracked_data(self):
             data = {}
             for f in self._meta.fields:
                 try:
-                    data[f.column] = self.__get_field_value(f)
+                    data[f.column] = copy(self.__get_field_value(f))
```

Comment:
What is actually returned from `__get_field_value(f)` here? Curious what this is that'd require a copy. And to make sure this will handle everything.

Reply:
`__get_field_value(f)` returns the value of immutable objects, but since bitfields are mutable, we're not able to compare on a reference point. For example, using the `has_changed()` method will always return `False` since we're comparing the same (changed) value.

Reply:
Sure, but what I'm saying is that copying all data here has potentially larger ramifications compared to not copying. Can we be selective on when we copy and when we don't copy? Like, can we control this to _only_ copy on bitfields and not other fields that don't need copied?

Reply:
Ah, understood - Updated to check if an immutable object.